roles/parse-docker-files/tasks/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+
+- name: Get a list of directory names in project
+  raw: ls ../
```

Comment:
might it not be cleaner to use the {{ playbook_dir }} variable as a basis, rather than ls